### PR TITLE
Html css linters config

### PR DESCRIPTION
**Pull request by Tamunokuro**
Add Linters file
Add HTML $ CSS file

### CODE REVIEW

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @Tamunokuro, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️

- [ ] Your project is close to perfect 😄 . Please extract the `.hintrc` and` .stylelintrc.json` from the `.githib` folder and add them to the root directory. Only the `linters.yml` should be in the `.github` folder

- [ ] Kindly make sure your readme is descriptive and follows the format used in Microverse which can be seen [here](https://github.com/microverseinc/readme-template) 🙏

_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -4,3 +4,14 @@ github flows,
...
 and my ability to submit a project for code review
+
+# Author:
+Joshua Blue-Jack
+linkedin: Joshua Blue-Jack
+github: https://github.com/Tamunokuro
+
+# Acknowldgement: 
+AbhishekkM05
+github: https://github.com/AbhishekkM05
```
Comment (Kamba56) [inline README.md:15]: - [ ] As requested by the previous code reviewer, please use the [readme template ](https://github.com/microverseinc/readme-template)provided by microverse for your readme. Only remove the sections that do not apply to your project such as the test section. Please add steps that one can take to get a local copy of your project under the `Getting Started` section. The steps can be;
- `clone the repo using the command git clone https://github.com/Tamunokuro/hello_microverse.git`
- `run npm install`
- etc

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @Tamunokuro, 

Great work on making the changes requested by a previous reviewer 👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [x] Great use of Github flow
- [ ] Meaningful commit messages
- [ ] Descriptive PR title and summary

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,17 +1,31 @@
...
+## Prerequisites
+- Code Editor
+- GIT
+- NODE PACKAGE MANAGER
+
+# Authors
+## 👤 Authors
+
+- Linkedin: (www.linkedin.com/in/joshua-blue-jack/)
+- Github: (https://github.com/Tamunokuro)
```
Comment (aimalamiri) [inline README.md:20]: [Optional] You can add links to your online profiles using this syntax `[Link Title](Link URL)`.

**Review by aimalamiri — APPROVED** [review]
Hi @Tamunokuro, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ApprovedDavieGIF


## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**